CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ set(CMAKE_C_STANDARD 11)
 include(GenerateExportHeader)
 if(MSVC)
     set(CMAKE_CXX_FLAGS "-W4 /permissive- /GS ${CMAKE_CXX_FLAGS}")
-    set(CMAKE_EXE_LINKER_FLAGS "/MANIFEST:NO /SUBSYSTEM:WINDOWS /ENTRY:mainCRTStartup ${CMAKE_EXE_LINKER_FLAGS}")
+    set(CMAKE_EXE_LINKER_FLAGS "/MANIFEST:NO /SUBSYSTEM:WINDOWS /ENTRY:mainCRTStartup /STACK:8388608 ${CMAKE_EXE_LINKER_FLAGS}")
```

Comment:
Can you add comments for what these do so we won't be lost in the future?

Reply:
Just the linker flags or the compiler flags aswell?

I had actually meant to remove the 'SUBSYSTEM' & 'ENTRY' flags as with `CMP0020 OLD` removed the linker can figure it out on its own, I'll do that a little bit later, along with removing another flag set in the build workflow